transport/BaseUdpEndpoint.cpp

```diff
@@ -66,7 +66,7 @@ BaseUdpEndpoint::BaseUdpEndpoint(const char* name,
     : _state(Endpoint::CLOSED),
       _name(name),
       _localPort(localPort),
-      _receiveJobs(jobManager, 16),
```

Comment:
We had a problem with this size. We need to figure out what is the reasonable values